.github/workflows/release.yml

```diff
@@ -3,15 +3,17 @@ name: .NET Build and Publish
 on:
   push:
     branches: ["main", "installer-test/**"]
-    tags: ["v3.*"] # Manual delivery on every 3.x tag
+    tags: ["v3.*.*"] # Manual delivery on every 3.x tag
```

Comment:
hmm, what is changing here as behavior?

Reply:
Some repos were using `v3.*`, others were using `v3.*.*`.

I wanted to align everything, so I picked the one that is least likely to lead to an invalid semver

Reply:
before. a tag of `v3.1` would match, and trigger..
now you need to do atleast two `.` e.g. `v3.1.0` or `v3.1.0-beta` etc...

Reply:
oki, make sense